doc/content/video.md

```diff
@@ -230,7 +230,7 @@ set("frame.video.samplerate",12)
 s = single("big_buck_bunny_720p_stereo.ogg")
 # This hack is necessary (for now) in order
```

Comment:
Is this part of doc still relevant? I don't understand why we would need that here.

Reply:
👍